locales/fr.lua

```diff
@@ -0,0 +1,32 @@
+local Translations = {
+    error = {
+        to_far_from_door = 'Vous êtes trop loin de la sonnette',
+        nobody_home = 'Il n\'y a personne à la maison..',
+        nobody_at_door = 'Il n\'y a personne à la porte...'
+    },
+    success = {
+        receive_apart = 'Vous avez un appartement',
+        changed_apart = 'Vous avez changé d\'appartement',
+    },
+    info = {
+        at_the_door = 'Quelqu\'un est à la porte !',
+    },
+    text = {
+        options = '[E] Options d\'appartement',
+        enter = 'Entrez dans l\'appartement',
+        ring_doorbell = 'Sonnette de porte',
```

Comment:
```suggestion
        ring_doorbell = 'Sonner à la porte',
```

Sounds better like this I think!